### PR TITLE
Display all examples in the tutorial index

### PR DESCRIPTION
**Pull request by eb8680**
Following #2668, this PR adds a number of examples to the tutorial index that are currently only visible on GitHub.

### CODE REVIEW

**Review by fritzo — APPROVED** [review]

**Comment by fritzo** [comment]
feel free to merge as soon as tests pass

File: tutorial/source/baseball.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Example: analyzing baseball stats with MCMC
+=======================================================
```
Comment (martinjankowiak) [inline tutorial/source/baseball.rst:2]: don't these lengths need to match exactly? not sure how this markdown works...

Comment (eb8680) [reply tutorial/source/baseball.rst:2]: I find `rst` confusing but it seems to render correctly for me locally.

Comment (fritzo) [reply tutorial/source/baseball.rst:2]: The bars need to be *at least* as long as the text.

**Pull request merged**